addon/globalPlugins/wintenObjs/__init__.py

```diff
@@ -323,9 +326,11 @@ def uiaDebugLogging(self, obj, event=None):
 			if not event:
 				event = "no event specified"
 			if event != "controllerFor":
-				log.debug("W10: UIA object name: %s, event: %s, app module: %s, automation Id: %s, class name: %s"%(obj.name, event, obj.appModule, automationID, className))
+				log.debug("W10: UIA object name: {0}, event: {1}, app module: {2}, automation Id: {3},
+					class name: {4}".format(obj.name, event, obj.appModule, automationID, className))
```

Comment:
For this one and the recorder for controller for, it might be better to use the original version, as this code path will be executed if and only if I and some user restart NVDA with debug logging enabled. Thanks.